mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -712,4 +735,32 @@
 
     <!-- Label for the checkbox to send the tab crash report to Mozilla -->
     <string name="crash_report_send_crash_label">Send crash report to Mozilla</string>
+
+    <!-- Tracking protection bottom sheet -->
+
+    <!-- Label for the total number of trackers blocked. %s represents the number of trackers blocked. -->
+    <string name="trackers_count_note">Trackers blocked since %s</string>
```

Comment:
Shouldn't there be a colon before the "since" and the placeholder? It seems kinda confusing to me this way, like if "since" represents time rather than a number?
Or perhaps the word "since" feels like it doesn't have to be there?
so something more natural might sound like, "Trackers blocked: %s"
CC @mcarare

Reply:
@Delphine Thanks for tagging me. That is actually the app install date. I have changed the description to match: https://github.com/mozilla-mobile/focus-android/pull/5137.
As a note, the UX design does not have a colon after "since"

Reply:
Perfect! thank you